src/test/munit/mysql/data-types-test-case.xml

```diff
@@ -62,7 +62,7 @@ CREATE TABLE data_types_table (
         </db:execute-ddl>
     </munit:after-suite>
 
-    <munit:test name="testDataTypes">
+    <munit:test name="testDataTypes" ignore="true">
```

Comment:
I won't want to delay this PR again by asking you to commit a comment on why we are ignoring this test, I think it'll be enough by posting a comment directly in the PR